src/model/engines/engines.nit

```diff
@@ -15,3 +15,4 @@ module engines
 
 import engine_base
 import pep8
+import nitc
```

Comment:
I talked with @privat about this, there's no need to move it anymore since the instanciation of the engine and the addition to the engine map is done within `debug` until we find a way to execute it safely within a docker container